apl-api2/src/main/resources/yaml/apollo-api-v2.yaml

```diff
@@ -623,587 +626,1649 @@ paths:
+          schema:
+            type: string
+            example: 'js'
+        - name: version
+          in: query
+          description: The language version
+          required: false
+          schema:
+            type: string
+            example: '0.1.1'
+      responses:
+        200:
+          description: successful operation
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ContractSpecResponse'
+        400:
+          $ref: '#/components/responses/BadRequest'
+        500:
+          $ref: '#/components/responses/ServerError'
+
+  /@@smc/asr/{module}/src:
+    get:
+      tags:
+        - smc
+        - asr
+      summary: Get the ASR module source.
+      description: Returns the source of the ASR module
+      operationId: getAsrModuleSource
+      parameters:
+        - name: module
+          in: path
+          description: The ASR module name
+          required: true
+          schema:
+            type: string
+            example: 'APL20BUY'
+        - name: language
+          in: query
+          description: The language name
+          required: false
+          schema:
+            type: string
+            example: 'js'
+        - name: version
+          in: query
+          description: The language version
+          required: false
+          schema:
+            type: string
+            example: '0.1.1'
+      responses:
+        200:
+          description: successful operation
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ModuleSourceResponse'
+        400:
+          $ref: '#/components/responses/BadRequest'
+        500:
+          $ref: '#/components/responses/ServerError'
+
+  /@@smc/asr/{module}/spec:
+    get:
+      tags:
+        - smc
+        - asr
+      summary: Get the ASR module specification.
+      description: Returns the specification of the ASR module
+      operationId: getAsrModuleFullSpec
+      parameters:
+        - name: module
+          in: path
+          description: The ASR module name
+          required: true
+          schema:
+            type: string
+            example: 'APL20BUY'
+        - name: language
+          in: query
+          description: The language name
+          required: false
+          schema:
+            type: string
+            example: 'js'
+        - name: version
+          in: query
+          description: The language version
+          required: false
+          schema:
+            type: string
+            example: '0.1.1'
+      responses:
+        200:
+          description: successful operation
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/AsrSpecResponse'
+        400:
+          $ref: '#/components/responses/BadRequest'
+        500:
+          $ref: '#/components/responses/ServerError'
+
+  /@@smc/debug/{address}:
```

Comment:
Probably it is a conversion, not a debug.

Reply:
fixed